Core/Strings/Names/Business.txt

```diff
@@ -6,35 +6,35 @@ Wuland
 Yootanu
 Vornax
 Ludeon
-Ultimate
-Superior
-Starlight
-Sunlight
+Ultiem
+Beter
+Sterrenlicht
+Zonlicht
 Greengrow
-Grassland
-Planetary
+Grasland
+Planetair
 Homestead
 Zarimax
 Albium
-General
-Meteor
+Algemeen
+Meteoor
 Terroid
 Terra
-Old School
+Ouderwets
 Nouveau
-New
-The Best
-Professional
+Nieuw
+De Besten
```

Comment:
Dit is het enige woord dat meervoud is, is dat de bedoeling? Moeilijk te zien in review zonder te weten welke woorden hierachter gebruikt kunnen worden.

Reply:
Dit zijn namen waarachter business types worden gezet (in NamerTraderGeneral.rulePack.rulesStrings):
`<li>r_name(p=4)->[businessname] [businesstype]</li>`
waar `businesstype` uit Core/Strings/Words/Nouns/BusinessTypes.txt wordt gehaald.

Het hangt er een beetje van af hoe je het wil lezen, maar als we het als een bijvoeglijk naamwoord zien dan krijgen we problemen met het lidwoord, want de zijdigheid van de business types heb ik achterwege gelaten.